src/js/utils/option-utils.jsx

```diff
@@ -155,6 +155,24 @@ export const debounceOrganizationsFetch = (waitTime, minSearchLength) =>
     }
   }, waitTime);
 
+export const debounceAllOrganizationsFetch = (waitTime, minSearchLength) =>
+  _.debounce((searchTerm, callback) => {
+    if (searchTerm && searchTerm.length >= minSearchLength) {
+      apiClient.get(`/openboxes/api/organizations?q=${searchTerm}`)
```

Comment:
I think you should parametrize roleType in the debouncedOrganizationFetch. Do it like `activityCodes` in `debouncedOrganizationFetch`, which by default will be `['ROLE_SUPPLIER']` and for the case when you want a all organization, then you can pass empty list.